native/.gitignore

```diff
@@ -0,0 +1,53 @@
+# OSX
```

Comment:
Kinda wondering if we need all of these ...hard to tell mind you 🤔 .

Reply:
These were all autogenerated, I'll remove the ones I know we won't need for sure.

Reply:
I've removed a couple of things, but I think it might be best to keep the rest in there until we know for sure what we don't need - particularly the XCode stuff, as I can see a lot of these files mysteriously appearing if we open the project in XCode.